web/app/scripts/views/dashboard/dashboard-partial.html

```diff
@@ -8,10 +8,10 @@
             </div>
         </div>
         <div class="block block-1x2 quick-map">
-            <div element-stats class="block-inner">
-                <driver-stepwise chart-data="ctl.records"></driver-stepwise>
+            <div class="block-inner">
+                <div class="map" leaflet-map recent-events></div>
             </div>
-                <h3>Recent Accidents
+                <h3>Accidents during the past two weeks
```

Comment:
Maybe use `{{ ctl.recordType.plural_label }}` here, so it could easily apply to multiple record types.